test/worldwide/field_data_consistency_test.rb

```diff
@@ -37,6 +37,21 @@ def sanity_check_file(file_name)
         value.keys.each do |key|
           assert_includes(permitted_keys, key)
         end
+
+        assert_instructional_informative_error_messages(value["errors"], file_name, key)
+      end
+    end
+
+    def assert_instructional_informative_error_messages(errors, file_name, field_key)
+      return if errors.nil?
+      return unless file_name.end_with?("en.yml")
```

Comment:
This enables `async` translations, while enforcing english as the base language that must exist.